content/blog/2019-01-03-ozunconf18.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: "ozunconf18"
+title: Continuing to Grow Community Together at ozunconf, 2018
+authors:
+  - name: Nicholas Tierney
+    url: https://www.njtierney.com
+    twitter: nj_tierney
+# twitterImg: img/blog-images/2019-01-03-ozunconf18/fun-photo.jpg
+date: 2019-01-03
+categories: blog
+topicid: 925
+tags:
+- r
+- community
+- software
+- packages
+- data-acess
+---
+
+[summary photo]
+
+In late November 2018, we ran the [third annual rOpenSci ozunconf](http://ozunconf18.ropensci.org). This is the sibling rOpenSci unconference, held in Australia. We ran the first ozunconf in 2016, [in Brisbane](http://auunconf.ropensci.org), the second in 2017 [we ran it in Melbourne](http://ozunconf17.ropensci.org).
+
+As usual, before the unconf, we started discussion on [github issue threads](https://github.com/ropensci/ozunconf18/issues), 
+and the excitement was building with the number of issues.
+
+The day before the unconf we ran "Day 0 training" - an afternoon explaining R packages and github. This training is run a little different to a typical course on R packages and github, in that the aim is to get people acquainted with what R packages and GitHub are, that they can learn the material, and that it is worthwhile to learn. We showed people how to create an R pacakge, set it up with version control with git, then put it on github, and share it with others. This afternoon course was developed last year by myself and [Roger Peng](https://twitter.com/rdpeng). This year, [Saras Windecker](https://twitter.com/smwindecker) stepped up to the plate and did an incredible job teaching. We also had a great group of helper instructors, which ended up creating the fantastic teaching ratio of two students for every helper. Everyone picked everything up super quickly, and this meant that we could hit the ground running on Day 1.
+
+[photo of day 0 training]
+
+Day 1 started with a bang, in the beautiful north arts building at the university of Melbourne, and dusted off the tried and tested rOpenSci icebreaker. We then moved on to choose projects. Everyone moved around the room, exploring options, and we settled into groups.
```

Comment:
Please link to post on ice breaker now that we've published it: `[rOpenSci icebreaker](https://ropensci.org/blog/2018/11/01/icebreaker/)`